Spinner/Spinner.js

```diff
@@ -143,17 +164,15 @@ const SpinnerBase = kind({
 		)
 	},
 
-	render: ({children, css, ...rest}) => {
+	render: ({css, ...rest}) => {
 		delete rest.transparent;
 
 		return (
 			<UiSpinnerBase
 				{...rest}
 				css={css}
 				component={SpinnerCore}
-			>
-				{children}
```

Comment:
I removed `children` since it is not currently supported by agate and is automatically handled by including it in `rest`.